src/main/java/de/tum/in/www1/artemis/domain/iris/settings/IrisChatSubSettings.java

```diff
@@ -34,6 +34,19 @@ public class IrisChatSubSettings extends IrisSubSettings {
     @Column(name = "rate_limit_timeframe_hours")
     private Integer rateLimitTimeframeHours;
 
+    @Nullable
+    @Column(name = "lecture_chat")
+    private Boolean lectureChat;
```

Comment:
2 things:
- The lecture slides are relevant for more than just the tutor chat, so this should be part of the IrisSettings directly, instead of being part of the chat settings
- I'd like this to be more descriptive to show that it enables/disables the lecture ingestions, so more like `enableLectureIngestion`